### PR TITLE
history changes

### PR DESCRIPTION
**Pull request by AndreeaDora**

### CODE REVIEW

File: verigreen-collector-webapp/src/main/webapp/controllers.js [hunk 1]
```diff
@@ -488,6 +491,14 @@ app.controller('ctrlRead', ['$scope','$filter','$http', 'ModalService', 'sharedP
                         }
               }).success(function(data) {
                         $scope.historyData = data;
+                        var i=0;
```
Comment (eitansch) [inline verigreen-collector-webapp/src/main/webapp/controllers.js]: Style:
var i = 0;

File: verigreen-collector-webapp/src/main/webapp/controllers.js [hunk 2]
```diff
@@ -488,6 +491,14 @@ app.controller('ctrlRead', ['$scope','$filter','$http', 'ModalService', 'sharedP
...
+                        for (var key in $scope.historyData) {
+                            $scope.historyValues = $scope.historyData[key];
+                            for (var d = 0, len = $scope.historyValues.length; d < len; d += 1) {
```
Comment (eitansch) [inline verigreen-collector-webapp/src/main/webapp/controllers.js]: Style:
Why d += 1 and two lines below i++; ?
Please make all +1 into ++

**Comment by eitansch** [comment]
There's no LA in the index.html file - please add it (as a  block)

File: verigreen-collector-webapp/src/main/webapp/controllers.js (hunk 1 above)
Comment (eitansch) [inline verigreen-collector-webapp/src/main/webapp/controllers.js]: Style:
var i = 0;

**Comment by AndreeaDora** [comment]
Closing pull request because the commits were squashed. A new pull request will be pushed.

**Pull request closed without merging**